operator-common/src/main/java/io/strimzi/operator/common/MicrometerMetricsProvider.java

```diff
@@ -27,6 +27,14 @@ public MicrometerMetricsProvider() {
         this.metrics = BackendRegistries.getDefaultNow();
     }
 
+    /**
+     * Constructor of the Micrometer metrics provider
```

Comment:
Might be worth adding that this is here mainly for test purposes?

Reply:
added